include/no-os/uart.h

```diff
@@ -0,0 +1,162 @@
+/***************************************************************************//**
```

Comment:
this file seems to have been deleted in a previous commit, only to readded it here, this is not ok